src/components/video/_macro.njk

```diff
@@ -0,0 +1,7 @@
+{%- macro onsVideo(params) -%}
+    <div class="video">
+        {% if params.youtubeUrl %}
+            <iframe class="video__iframe" width="560" height="315" src="{{ params.youtubeUrl }}" frameborder="0" allow="accelerometer; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Should the width and height on this be hardcoded?

Reply:
Well it's what Youtube give you but it's overridden with css.

Reply:
If it's overridden it's probably best to remove it